examples/timer_periodic_wakeup/README:md

```diff
@@ -0,0 +1,44 @@
+# Periodic Wakeup Timer Example
+
+This example demonstrates how to set up a periodic wakeup timer using the RIOT operating system.
+
+## Overview
+
+The application sets up a timer that periodically wakes up the system and performs a specific task. This is useful for applications that need to perform regular actions, such as sensor readings or data transmissions.
+
+## Requirements
+
+- RIOT OS
+- A supported hardware platform
+
+## Setup
```

Comment:
This thorough step-by-step guide could in the future serve as a template for (automatically generated?) documentation about the examples at doc.riot-os.org. Feels a bit wordy compared to the other READMEs in the repo, but since they are all differing in detail anyway, I don't mind it too much.